packages/chain-adapters/src/ChainAdapterCLI.ts

```diff
@@ -172,11 +178,16 @@ const main = async () => {
     const cosmosAccount = await cosmosChainAdapter.getAccount(cosmosAddress)
     console.log(cosmosAccount)
 
-    // await cosmosChainAdapter.subscribeTxs(
-    //   { wallet, bip44Params: cosmosBip44Params },
-    //   (msg) => console.log(msg),
-    //   (err) => console.log(err)
-    // )
+    const cosmosTxHistory = await cosmosChainAdapter.getTxHistory({
+      pubkey: 'cosmos1zjk9dkhzz2waxmtvtl3hnnl0t3ac0k5urlyk7s'
```

Comment:
default to derived cosmos address
```suggestion
      pubkey: cosmosAddress
```